x-pack/plugins/apm/public/components/app/error_group_details/error_sampler/error_sample_detail.tsx

```diff
@@ -207,6 +230,39 @@ export function ErrorSampleDetails({
             </EuiLink>
           </EuiFlexItem>
         )}
+        {externalContextMenuItems.value?.length ? (
+          <EuiPopover
+            id="errorContextMenu"
+            button={
+              <EuiButton
+                data-test-subj="ErrorSampleDetailsButton"
+                onClick={() => setIsContextMenuOpen((isOpen) => !isOpen)}
+                iconType="arrowDown"
+                iconSide="right"
+              >
+                {i18n.translate(
+                  'xpack.pam.errorSampleDetails.investigateMenu',
+                  { defaultMessage: 'Investigate' }
+                )}
+              </EuiButton>
+            }
+            isOpen={isContextMenuOpen}
+            closePopover={() => setIsContextMenuOpen(() => false)}
+            panelPaddingSize="none"
+            anchorPosition="downLeft"
+          >
+            <EuiContextMenu
+              initialPanelId="mainMenu"
+              panels={[
+                {
+                  id: 'mainMenu',
+                  items:
+                    externalContextMenuItems.value as EuiContextMenuPanelItemDescriptor[],
+                },
+              ]}
+            />
+          </EuiPopover>
+        ) : undefined}
```

Comment:
Can you extract this (so it's easier to delete in the future since this is an experiment)

```suggestion
        <UiActionsContextMenu />
```

This way you can also encapsulate `externalContextMenuItems` and the `useState` hook